person.rb

```diff
@@ -1,26 +1,31 @@
 require_relative './name_able'
 require_relative './trimmer_decorator '
 require_relative './capital_decorator'
+require './rental'
 
 class Person < Nameable
-  attr_accessor :name, :age, :id
+  attr_accessor :age, :name, :id, :rentals
   attr_reader :parent_permission
```

Comment:
- Hey :wave: adding the parent_permission to the attr_acessor is not required here as the age, name and parent_permission are all shared entities for both persons (student and teacher). you only need the id as the attr_reader. Please, kindly update it to be the id. `attr_reader :id`.